.github/workflows/BuildAOSMWheel.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches:
+      - add-aosm-extension
+
+jobs:
+  build_aosm:
+    runs-on: ubuntu-latest
+    container: mcr.microsoft.com/azure-cli/tools:latest
+    permissions: write-all
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: Build AOSM Wheel
```

Comment:
Could you please tell why we need this github action to build wheel? Our CI will build wheel file for your extension automatically